src/main/scala/com/johnsnowlabs/ml/tensorflow/TensorflowWrapper.scala

```diff
@@ -96,14 +96,28 @@ object TensorflowWrapper {
     else
       file
 
+    //Use CPU
+    //val config = Array[Byte](10, 7, 10, 3, 67, 80, 85, 16, 0)
+    //Use GPU
+    val config = Array[Byte](56, 1, 64, 1)
+
     // 3. Read file as SavedModelBundle
-    val graphDef = Files.readAllBytes(Paths.get(folder, "saved_model.pb"))
-    val graph = new Graph()
-    graph.importGraphDef(graphDef)
-    val session = new Session(graph)
-    session.runner.addTarget("save/restore_all")
-      .feed("save/Const", t.createTensor(Paths.get(folder, "variables").toString))
-      .run()
+    val (graph, session) = if (useBundle) {
+      val model = SavedModelBundle.load(folder, tags:_*)
```

Comment:
Curiosity question: any particular advantage to using this? best integration with Python maybe?

Reply:
Pros: Use Tensorflow API for loading model
Cons: There is no Save API for ModelBundle in JAVA, so we cannot train using this. Using SavedModelBundle is only useful for reading complete models trained in python. Otherwise, we need to use Aleksei's decomposed loading method to be able to save it manually during training (using python provided graph)